tests/layer_tests/common/layer_test_class.py

```diff
@@ -177,9 +177,12 @@ def compare_ie_results_with_framework(self, infer_res, framework_res, framework_
                             atol=framework_eps,
                             rtol=framework_eps):
                 is_ok = False
-                print("Max diff is {}".format(
-                    np.array(
-                        abs(infer_res[ie_out_name] - framework_res[framework_out_name])).max()))
+                if ie_res.dtype != bool:
+                    print("Max diff is {}".format(
+                        np.array(
+                            abs(infer_res[ie_out_name] - framework_res[framework_out_name])).max()))
```

Comment:
Either this should be fixed or we need to remove ie_res entirely -- seems like output naming is always right.
```suggestion
                            abs(ie_res - framework_res[framework_out_name])).max()))
```